xmake/rules/qt/load.lua

```diff
@@ -141,7 +141,7 @@ function main(target, opt)
     local languages = target:get("languages")
     local cxxlang = false
     for _, lang in ipairs(languages) do
-        if lang:startswith("xx") or lang:startswith("++") then
+        if lang:startswith("cxx") or lang:startswith("c++") then
```

Comment:
This is to implement lookup support for gnuc++/gnucxx. maybe we should use find instead of startswith.

```lua
if lang:find("cxx", 1, true) or lang:find("c++", 1, true) then
```

Reply:
In that case I would do
```lua
if lang:startswith("cxx") or lang:startswith("c++") or lang:startswith("gnucxx") or lang:startswith("gnuc++") then
```
instead, but find should work as well at least until some weird `-std` value is introduced.

Reply:
This is a bit bloated, and `find("-std", 1, true)` only find plain text, it doesn't handle pattern matching.